Make it easier to find tyk demos (#4030)
TykTechnologies/tyk-docs: tyk-docs/content/getting-started/quick-start.md

---

> Title: Quick Start (Free Trial)

Author: ```suggestion
Title: Quick Start - Free Trial
```

weight: 1
---

New to *Tyk Self Managed*? Click the button below to kickstart your journey with the basics.

**Important note:** Upon clicking, you will need to sign up. Take advantage of our **2-week trial** for the *Tyk Dashboard & Developer Portal*, and receive your temporary license along with installation instructions during the process.

{{< button_left href="https://tyk.io/sign-up#self" color="green" content="Free trial" >}}

## What’s Included in the Quick Start?
- View an example API secured through the Tyk Gateway, configured with rate limiting.
- Login to Tyk Dashboard to view API analytics and update the configuration of the API (e.g. with further authentication options).
- Update rate limits for the example API.

## Trial Prerequisites

> The recommended way for starting your free trial is via *Docker*. Further instructions for installation docker are available [here](https://docs.docker.com/get-docker/).

Author: actually isn't it "docker installation" ?

> The recommended way for starting your free trial is via *Docker*. Further instructions for installation docker are available [here](https://docs.docker.com/get-docker/).

Author: ```suggestion
The recommended way to start your free trial is via *Docker*. Follow the [instructions](https://docs.docker.com/get-docker/) to install *Docker*.
```
